internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -1200,3 +1200,143 @@ func (s *Server) Memory(ctx context.Context, in *empty.Empty) (reply *machine.Me
 
 	return reply, err
 }
+
+// EtcdMemberList implements the machine.MachineServer interface.
```

Comment:
All of these methods should use a common package. Some of this code is copied from the tasks file.